.github/workflows/build.yml

```diff
@@ -528,6 +528,7 @@ jobs:
     needs: [job_get_metadata, job_build]
     if: needs.job_get_metadata.outputs.changed_browser_integration == 'true' || github.event_name != 'pull_request'
     runs-on: ubuntu-20.04
+    timeout-minutes: 10
```

Comment:
I'm probably missing something but aren't we already pretty close to 10 minutes? Just wondering if this is maybe a little too tight, wdyt?

Reply:
yeah, you're right, it varies a lot (sometimes 5min, sometimes 9min, ...) - I'll bump it to 15min!